app/controllers/v0/health_records_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module V0
+  class HealthRecordsController < BBController
+    def refresh
+      resource = client.get_extract_status
+
+      render json: resource.data,
+             serializer: CollectionSerializer,
+             each_serializer: ExtractStatusSerializer,
+             meta: resource.metadata
+    end
+
+    def eligible_data_classes
+      resource = client.get_eligible_data_classes
+
+      render json: resource,
+             serializer: EligibleDataClassesSerializer,
+             meta: resource.metadata
+    end
+
+    def create
+      client.post_generate(params)
+
+      render nothing: true, status: :accepted
+    end
+
+    def show
+      resource = client.get_download_report('pdf')
```

Comment:
Looking at `BB::Client`: "It should be quick enough that download report can be called more or less right after". What is behavior if it can't be called (caller is really fast or MHV is slow for some reason). I would guess get_download_report returns a 404 or something - should make sure frontend team is aware of this and thinks through UX implications.